CVLByExample/ReadOnlyReentrancy/contracts/VulnerableBankOther.sol

```diff
@@ -0,0 +1,30 @@
+import "@openzeppelin/contracts/utils/math/SafeMath.sol";
+
+contract VulnerableBankOther {
+  
+    using SafeMath for uint256;
+    mapping(address => uint) public balances;
+
+    function donate(address _to) public payable {
+        balances[_to] = balances[_to].add(msg.value);
+    }
+
+    function balanceOf(address _who) public view returns (uint balance) {
+        return balances[_who];
+    }
+
+    function getEthBalance() external view returns(uint){
+        return address(this).balance;
+    }  
+    function withdraw(uint _amount) public {
+        if (balances[msg.sender] >= _amount) {
+            (bool result,) = msg.sender.call{value:_amount}("");
+            if (result) {
+                _amount;
```

Comment:
What does this line (_amount) do?

Reply:
actually this is the original code, I did not touch it. It's pretty uncommon https://stackoverflow.com/questions/70767898/what-does-mean-expression-variablename-in-solidity
I even found a stack overflow issue about it... 
I can fix it or not, wdyt?